docs/preprocessor/preprocess-file.sh

```diff
@@ -139,7 +140,7 @@ if [ ! ${SKIP} ] && [ $(grep -c '^\[gremlin' ${input}) -gt 0 ]; then
   ${lb} awk -f ${AWK_SCRIPTS}/language-variants.awk > ${output}
 
   ps=(${PIPESTATUS[@]})
-  for i in {0..6}; do
+  for i in {0..7}; do
```

Comment:
I'm not able to follow why this is a loop from 0 to 7... can we include a comment or move `7` to a constant with a name?

Reply:
Ha, it should actually be 0 to 9, as it checks to exit code for each program in the pipe chain. Not sure when that went out of sync, but I'll fix it.